PROBot/UserSettings.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System.IO;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Serialization;
+
+namespace PROBot
+{
+    public static class UserSettings
```

Comment:
The class doesn't need to be `static`, you can instantiate it in `BotClient` or in the view.
That way we can instantiate multiple clients with different settings if we want to.